Bottom: 5708b36827a9934ecc2eb8e152edf713492f1e7d
Top:    d1c12cb4fb9aadc23d2fcdbdfe42f9fd03db3945
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:14 +0000

Fix undo in MoveObjectAndRubberband

XXX: Need to check if the path which previously missed
     IncrementUndoSerialNumber() was doing so for a reason.


---

diff --git a/src/move.c b/src/move.c
index cfdf881..15cbddc 100644
--- a/src/move.c
+++ b/src/move.c
@@ -946,6 +946,7 @@ MoveObjectAndRubberband (int Type, void *Ptr1, void *Ptr2, void *Ptr3,
       MoveLinePoint (Ptr1, moving_line, &moving_line->Point2);
 
       ptr2 = moving_line;
+      IncrementUndoSerialNumber ();
     }
   else
     {
